scripts/gen_vimdoc.py

```diff
@@ -190,17 +248,20 @@
 # Raises an error with details about `o`, if `cond` is in object `o`,
 # or if `cond()` is callable and returns True.
 def debug_this(cond, o):
-    name = ''
+    name = ""
     if not isinstance(o, str):
         try:
             name = o.nodeName
-            o = o.toprettyxml(indent='  ', newl='\n')
+            o = o.toprettyxml(indent="  ", newl="\n")
         except Exception:
             pass
-    if ((callable(cond) and cond())
-            or (not callable(cond) and cond)
-            or (not callable(cond) and cond in o)):
-        raise RuntimeError('xxx: {}\n{}'.format(name, o))
+
+    if (
+        (callable(cond) and cond())
+        or (not callable(cond) and cond)
+        or (not callable(cond) and cond in o)
+    ):
+        raise RuntimeError("xxx: {}\n{}".format(name, o))
```

Comment:
what formatter did you use? would be better to avoid formatting changes, or let's standardize on Black?